deployments/mumbai/usdc/configuration.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenPriceFeed": "0x572dDec9087154dC5dfBB1546Bb62713147e0Ab0",
+  "borrowMin": "1000e6",
```

Comment:
I think this should be 100 like mainnet unless we're setting it higher on purpose

Reply:
sounds good. updated